keycloak/themes/haven/account/resources/js/template.ftl

```diff
@@ -0,0 +1,131 @@
+<#macro mainLayout active bodyClass>

```

Comment:
@knmurphy do you know why this file is added? thinking it might be a mistake

Reply:
@statik thats a mistake, removed